packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -7,10 +7,13 @@ import { toTitleCase } from '../../data-helper';
 /**
  * Type dependencies
  */
-import { Page } from 'playwright';
+import { ElementHandle, Page } from 'playwright';
 
 const selectors = {
 	sidebar: '.sidebar',
+	heading: '.sidebar > li:not(.sidebar__menu-item--child)',
```

Comment:
I think the `:not` part is not necessary here, because the `>` will select only the immediate `li` descendants of the `.sidebar` element.
```suggestion
	heading: '.sidebar > li',
```

Reply:
Looks like you are correct; verified in Playwright debugger:
`playwright.$${'.sidebar > li span:has-text("Settings"):visible')`

selected only the top-level `Settings` as I expected.